src/javascript/app/base/binary_pages.js

```diff
@@ -112,7 +112,7 @@ const pages_config = {
     trading                  : { module: TradePage,                  needs_currency: true },
     transferws               : { module: PaymentAgentTransfer,       is_authenticated: true, only_real: true },
     two_factor_authentication: { module: TwoFactorAuthentication,    is_authenticated: true },
-    virtualws                : { module: VirtualAccOpening,          not_authenticated: true },
+    virtualws                : { module: VirtualAccOpening,          only_non_authenticated: true },
```

Comment:
why are we not using `not_authenticated` as before? this way it handles it for other pages too, such as `new-account`, which is what we wanted